plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -307,44 +305,26 @@ public static Optional<MessageType> getParquetMessageType(List<HiveColumnHandle>
         return message;
     }
 
-    public static Optional<org.apache.parquet.schema.Type> getParquetType(GroupType groupType, boolean useParquetColumnNames, HiveColumnHandle column)
```

Comment:
sure, I would like reorganize the commits once the PR had been finalized, to avoid back and forth editing the commits. Is this plan sounds good to you?

Reply:
usually it helps with reviews/iteration if the core changes are isolated in separate commits. But seems okay doing that later for this one if that helps

Reply:
+1 
I would highly recommend commit separation as early as possible to aid reviewers. It brings some overhead to the PR author as you might need to rebase commits after addressing review comments. But it's worth it for enabling easier iteration in code reviews.

Reply:
thanks, I found more commands at: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History will follow and use those.